tests/Integration/Monitoring/Test_Plotter.py

```diff
@@ -1,416 +0,0 @@
-    assert res == 0.0
-
-
-def test_plotMaxFD():
-    """test for the method "_plotMaxFD" """
-
-    plotName = "ComponentMonitoringPlotter_plotMaxFD"
-    reportRequest = {
-        "grouping": "component",
-        "groupingFields": ("%s", ["component"]),
-        "startTime": 1562796000,
-        "endTime": 1562882400,
-        "condDict": {"component": "Monitoring_Monitoring"},
-    }
-
-    plotInfo = {
-        "data": {
-            "Monitoring_Monitoring": {
-                1562853600: 0,
-                1562857200: 1.0,
-                1562861700: 2.5,
-                1562854500: 4.4,
-                1562855400: 2.3333333333333335,
-                1562856300: 2.076923076923077,
-                1562858100: 1.3333333333333333,
-                1562849100: 0,
-                1562860800: 3.4615384615384617,
-                1562850000: 0,
-                1562848200: 0,
-                1562859900: 0,
-                1562850900: 0,
-                1562851800: 0,
-                1562847300: 0,
-                1562852700: 0,
-            }
-        },
-        "unit": "MaxFD",
-        "granularity": 900,
-    }
-
-    obj = ComponentMonitoringPlotter(None, None)
-    res = obj._plotMaxFD(reportRequest, plotInfo, plotName)
-    assert res["OK"] is True
-    assert res["Value"] == {"plot": True, "thumbnail": False}
-
-    res = compare("%s.png" % plotName, "%s.png" % os.path.join(plots_directory, plotName))
-    assert res == 0.0
-
-
-def test_plotActivityRunningThreads():
-    """test for the method "_plotActivityRunningThreads" """
-
-    plotName = "ComponentMonitoringPlotter_plotActivityRunningThreads"
-    reportRequest = {
-        "grouping": "component",
-        "groupingFields": ("%s", ["component"]),
-        "startTime": 1562796000,
-        "endTime": 1562882400,
-        "condDict": {"component": "Monitoring_Monitoring"},
-    }
-
-    plotInfo = {
-        "data": {
-            "Monitoring_Monitoring": {
-                1562853600: 0,
-                1562857200: 6.866666666666666,
-                1562861700: 6.0,
-                1562854500: 5.6,
-                1562855400: 6.2,
-                1562856300: 6.538461538461538,
-                1562858100: 6.333333333333333,
-                1562849100: 0,
-                1562860800: 6.153846153846154,
-                1562850000: 0,
-                1562848200: 0,
-                1562859900: 6.2,
-                1562850900: 0,
-                1562851800: 0,
-                1562847300: 0,
-                1562852700: 0,
-            }
-        },
-        "unit": "RunningThreads",
-        "granularity": 900,
-    }
-
-    obj = ComponentMonitoringPlotter(None, None)
-    res = obj._plotActivityRunningThreads(reportRequest, plotInfo, plotName)
-    assert res["OK"] is True
-    assert res["Value"] == {"plot": True, "thumbnail": False}
-
-    res = compare("%s.png" % plotName, "%s.png" % os.path.join(plots_directory, plotName))
```

Comment:
There are some images in tests/Integration/monitoring/plots that should be also removed.